hw/dv/data/sim.mk

```diff
@@ -4,6 +4,9 @@
 
 .DEFAULT_GOAL := all
 
+LOCK_SW_BUILD   ?= lockfile -1 -l 3600 ${sw_build_dir}/lock
+UNLOCK_SW_BUILD ?= rm -rf ${sw_build_dir}/lock
```

Comment:
Why `-rf`? The created lockfile is a normal file isn't it?

Reply:
This part is now rewritten.